tests/test.rs

```diff
@@ -398,6 +398,6 @@ fn bytes() {
     let data = b"abc\0123";
     let s = serialize(&data, Infinite).unwrap();
     let s2 = serialize(&Bytes::new(data), Infinite).unwrap();
-    assert_eq!(s, s2);
+    assert_ne!(s, s2);
```

Comment:
So I'm looking at the tests we already have. We do fixed sized array tests so just doing `assert_eq!(s, vec![ byte values here ])` wouldn't really add anything to the test coverage. What I could do is `assert_eq!(s[..], s2[8..]);` to verify that after the length encoding they are identical.